src/trait_manager/index.ts

```diff
@@ -62,12 +62,9 @@ export default class TraitManager extends Module<TraitManagerConfig & { pStylePr
    */
   constructor(em: EditorModel) {
     super(em, 'TraitManager', defaults);
-    const c = this.config;
     const model = new Model();
     this.model = model;
-    const ppfx = c.pStylePrefix;
-    this.types = { ...typesDef };
-    ppfx && (c.stylePrefix = `${ppfx}${c.stylePrefix}`);
+    this.types = typesDef;
```

Comment:
Looks all legit, the only issue I see is with the removal of the that thing for `stylePrefix` which makes them render incorrectly. 
<img width="234" alt="traits" src="https://user-images.githubusercontent.com/11614725/227147561-179cd82e-3b48-4825-bc31-207b723c5bb4.png">

Don't you see them in the same way?

Reply:
Yeah thanks for catching it I didn't realised that it changed, I updated the fix for it.